lib/cartopy/feature/__init__.py

```diff
@@ -251,6 +253,8 @@ def __init__(self, category, name, scale, **kwargs):
             The dataset scale, i.e. one of '10m', '50m', or '110m',
             or Scaler object. Dataset scales correspond to 1:10,000,000,
             1:50,000,000, and 1:110,000,000 respectively.
+        version: optional
```

Comment:
Can you also update the above cases to follow numpydoc formatting by adding ` : str` to all of them.
```suggestion
        version : str, optional
```